improver/cli/__init__.py

```diff
@@ -332,6 +340,9 @@ def with_output(
         output (str, optional):
             Output file name. If not supplied, the output object will be
             printed instead.
+        tee (bool):
+            Pass through the output object even if saved to file.
+            Used in pipelines of commands if intermediate output needs to be saved.
```

Comment:
This will a useful feature to have (whether or not it could be done asynchronously)